libs/api/domains/auth/src/lib/resolvers/delegation.resolver.ts

```diff
@@ -22,12 +22,12 @@ import {
   PatchDelegationInput,
   UpdateDelegationInput,
 } from '../dto'
-import { Delegation } from '../models'
+import { Delegation, MergedDelegation } from '../models'
 import { ActorDelegationsService } from '../services/actorDelegations.service'
 import { ActorDelegationInput } from '../dto/actorDelegation.input'
 import { MeDelegationsService } from '../services/meDelegations.service'
-import type { DelegationDTO, MergedDelegationDTO } from '../services/types'
-import { MergedDelegation } from '../models/delegation.model'
+import type { DelegationDTO } from '@island.is/clients/auth/delegation-api'
+import type { MergedDelegationDTO } from '@island.is/clients/auth/public-api'
```

Comment:
~ We prefer to have the import sorted into groups as

```
npm packages

monorepo module imports (@island.is/.... )

app/library relative imports

css imports.
```
But we currently need to manually do it, we need to add automatic ESLint/Prettier rules and formatting for this.